reinforcement_learning/rlclientlib/console_tracer.cc

```diff
@@ -0,0 +1,8 @@
+#include "console_tracer.h"
+#include <iostream>
+
+namespace reinforcement_learning {
+  void console_tracer::log(const std::string & msg) {
+    std::cout << msg << std::endl;
```

Comment:
>std::cout << msg << std::endl; [](start = 4, length = 30)

Do we want to constrain ourselves to cout? I think it would be very beneficial to do this an ostream logger that way we can log to cout, cerr or a file on disk. This would make it a lot simpler for us to collect logs from a customer.

Reply:
Good point.  I don't see a clean way of doing that without passing and ostream to live_model constructor or changing the factory signature to take an ostream.  Will have to address this in another PR.